cactusbot/handlers/spam.py

```diff
@@ -0,0 +1,59 @@
+"""Handle incoming spam messages."""
+
+from ..handler import Handler
+from ..packets import BanPacket, MessagePacket
+
+
+class SpamHandler(Handler):
```

Comment:
Very happy with how this turned out - infinitely better than what we had in v0.3. :tada: